### DISCUSSION TITLE
Trying to install via AltStore completely removes AltStore from device.

### DISCUSSION THREAD

**Discussion by whibber** [qnblackcat/uYouPlus#404] [General]
As the title states, every time I try installing uYouPlus through AltStore, it ends up freezing my AltStore app. I'll close out of the app and try to go back in but I get the "AltStore is no longer available" message, forcing me to reinstall. It might be a compatibility issue with iOS beta 16.3+ as it only started after updating.

**Comment by S3bbi96** [comment]
Had the same issue, is on IPadOS 16.1. Had to use sideloady to get it to work.

**Reply by tresenblanco** [reply]
Same issue iOS 16 beta 7 I fix by just letting the phone installing the ipa through AltStore without letting the phone or the computer turning off their screens
